src/odemis/util/angleres.py

```diff
@@ -161,6 +160,16 @@ def _FindAngle(x_array, y_array, pixel_size, parabola_f):
     return theta, phi, omega
 
 
+def _flipDataIfMirrorFlipped(data):
+    # invert data and adjust metadata for inverted mirror
+    data = data[::-1, :]
```

Comment:
Please move the `if focus_distance ...` block here too.

Reply:
Done. I also thought of it but it seems quite redundant to me. In case the condition is not true, the whole function _flipDataIfMirrorFlipped just checks the condition without any functionality.